ruby.html.markdown

```diff
@@ -215,6 +218,17 @@ else
   'else, also optional'
 end
 
+unless false
+  'true statement'
+else
+  'false statement, optional'
+end
+
+'if statement' if true
+'else statement' unless true
+
+(condition) ? 'true statement' : 'false statement'
+
```

Comment:
I would change this entire part to `puts 'true statement'`, ect, as it gets the point across better the purpose of these conditionals. Furthermore, it is bad practice for a conditional to be `unless`/`else`, but rather `if true`/`else`.

Also, include a comment explaining how line 230 is uses the ternary operator.